extensions/PhabBugz/lib/Feed.pm

```diff
@@ -84,7 +84,7 @@ sub feed_query {
         $self->logger->debug("STORY PHID: $story_phid");
         $self->logger->debug("AUTHOR PHID: $author_phid");
         $self->logger->debug("OBJECT PHID: $object_phid");
-        $self->logger->debug("STORY TEXT: $story_text");
+        $self->logger->info("STORY TEXT: $story_text");
```

Comment:
And I assume this is single-line or else the logger handles multiple lines. LGTM.

Reply:
The story text is always a single line.